### PR TITLE
Documentation cleanup

### PR DESCRIPTION
**Pull request by FTCHD**
Drizzle is my favorite ORM now, so here's a few tweaks to the docs!

## Changes
- Improved readability, consistency, tone and ease of reading.
- Fixed missing/wrong links.
- Add missing sentence ending dots.
- Changed - to — where it made sense (dash is easier on the eyes).
- Add `` where it made sense.
- Emphasized links for better visibility (no style changes).
- Upgraded the folder schema inline text graphic to native nextra `FileTree` component (in _Schema > Overview_).
- Transformed elements to native Nextra elements (`Callout`, `Cards`, `Tabs`).
- Added table declaration info in SQLite section.
- Emojis ✨

## Fixes
- #100 
- https://github.com/drizzle-team/drizzle-orm/issues/1128
- https://github.com/drizzle-team/drizzle-orm/issues/1212
- https://github.com/drizzle-team/drizzle-orm/issues/1021
- https://github.com/drizzle-team/drizzle-orm/issues/815 (temporary clarification)
- https://github.com/drizzle-team/drizzle-orm/issues/935

### CODE REVIEW

File: pages/docs/overview.mdx [hunk 1]
```diff
@@ -1,48 +1,51 @@
...
+First and foremost, Drizzle is a library and a collection of complementary opt-in tools. 
+
+**ORM** stands for _object relational mapping_, and developers tend to call Django-like or Spring-like tools an ORM. 
+We trully believe it's a misconception based on legacy nomenclature and we call them **data frameworks**.
 
-With data frameworks you have to build projects **around them** and not **with them**.  
-Drizzle lets you build your project the way you want or need, lets you be productive and does not interfere. 
-It lets you solve SQL problems with SQL toolkit and provide you a variety of opt in tools 
-to take your developer experience through the roof.  
+<Callout type="error" emoji="️💔">
```
Comment (darvesh) [inline pages/docs/overview.mdx:27]: image

Comment (FTCHD) [reply pages/docs/overview.mdx:27]: Could you share more info? 

Not seeing this behavior on any browser I tested on or online screenshoting services.

https://pikwy.com/web/6504c434b5cdfb554b2758f2

screenshot

**Comment by AlexBlokh** [comment]
I'm blown away by your PR, thank you so much
Will go through today

File: pages/docs/overview.mdx (hunk 1 above)
Comment (darvesh) [reply pages/docs/overview.mdx:27]: @FTCHD Apologies for not adding more information. I'm on Firefox 117.0, MacBook Pro 13inch.
image

**Comment by FTCHD** [comment]
Well, they felt like you had an ORM to deliver first, so I had to polish them a bit.

Hope it's per standard, and no worries if not just let me know!

**Review by AlexBlokh — CHANGES_REQUESTED** [review]
Can you please fix conflicts and I'll gladly merge everything

File: pages/kit-docs/commands.mdx [hunk 2]
```diff
@@ -314,3 +317,7 @@ drizzle-kit studio
 
 drizzle-kit studio --port 3000 --verbose
 ```
+
+<Button as="a">
+  <a href="http://0.0.0.0:3000">🖥 Open Drizzle Studio</a>
```
Comment (AlexBlokh) [inline pages/kit-docs/commands.mdx:322]: it should either open https://demo.drizzle.studio or we now have https://local.drizzle.studio

we should potentially have 2 and first button should be `open demo drizzle studio`

File: pages/kit-docs/overview.mdx [hunk 3]
```diff
@@ -262,15 +267,23 @@ $ drizzle-kit introspect:mysql ...
...
 Studio requires drizzle config file with `schema` and `dbCredentials` provided.
 ```bash
 drizzle-kit studio
 
 drizzle-kit studio --port 3000 ## custom port
 drizzle-kit studio --verbose   ## log all sql statements
 ```
+
+<Button as="a">
+  <a href="http://0.0.0.0:3000">🖥 Open Drizzle Studio</a>
```
Comment (AlexBlokh) [inline pages/kit-docs/overview.mdx:287]: Same here

File: pages/docs/sql-schema-declaration.mdx [hunk 4]
```diff
@@ -1,44 +1,67 @@
...
-    ├  ...
-```
-
-You can declare tables, indexes and constraints, foreign keys and enums.  
-Please pay attention to `export` keyword, they are mandatory if you'll be using [drizzle-kit SQL migrations generator](/docs/migrations)
+You can declare your SQL schema directly in TypeScript either in a single `schema.ts` file,
+or you can spread them around — whichever you prefer, all the freedom!
+
+
+<Tabs items={['1 File', 'Separate Files', 'Separate Folders']}>
```
Comment (AlexBlokh) [inline pages/docs/sql-schema-declaration.mdx:9]: We've discussed this internally and we don't wanna introduce complexity here by having `FileTree`, md text works completely fine, let's stick to it

I'm not sure about tabs though, having them makes things compact and my only concern is we lose explicit "you see everything on the screen" straight away

Comment (AlexBlokh) [reply pages/docs/sql-schema-declaration.mdx:9]: Let's remove FileTree and see how it looks with tabs
I think side by side 3 elements could be the best solution here

**Review by AlexBlokh — CHANGES_REQUESTED** [review]

**Comment by jasperkelder** [comment]
Awesome work @FTCHD! While going through the docs I was thinking of contributing some improvements myself. I'll wait for this to go live for now, thanks for the effort.

**Comment by FTCHD** [comment]
> Awesome work @FTCHD! While going through the docs I was thinking of contributing some improvements myself. I'll wait for this to go live for now, thanks for the effort.

Thx for hyping it up haha 😆

I'm using it myself instead of the live docs because I always have to manually search to see the SQL table declaration example.


Screenshot 2023-10-27 at 12 07 00 PM
Screenshot 2023-10-27 at 12 06 33 PM



Anyways, team might be busy (saw one of the guys go to Ukraine, my heart goes out to him we're not that far actually. I hope he knows what he's doing. This pkg is a beast and it needs their keepers) or maybe they want me to sit down and be more of a user rather than a commiter :?



Will keep resolving conflicts till then.

File: pages/docs/quick-sqlite/better-sqlite3.mdx [hunk 5]
```diff
@@ -45,13 +45,30 @@ const db = drizzle(sqlite);
...
+
+<Callout type="info" emoji="ℹ️">
+  If your `db.insert()` returns data, append `all()` to your query, otherwise append `run()`
+</Callout>
+
 If you want to use **sync** APIs:
 ```typescript copy
-import { drizzle } from 'drizzle-orm/bun-sqlite';
-import { Database } from 'bun:sqlite';
+import { drizzle, BetterSQLite3Database } from 'drizzle-orm/bun-sqlite';
```
Comment (AlexBlokh) [inline pages/docs/quick-sqlite/better-sqlite3.mdx:67]: we've removed type annotations of the database across the docs, I'll merge the pr and remove them myself

**Review by AlexBlokh — APPROVED** [review]

**Comment by AlexBlokh** [comment]
Thank you so much, @FTCHD!

@jasperkelder you're ready to go

**Pull request merged**